x-pack/plugins/triggers_actions_ui/public/application/components/delete_modal_confirmation.tsx

```diff
@@ -6,39 +6,51 @@
 import { EuiConfirmModal, EuiOverlayMask } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import React from 'react';
+import { HttpSetup } from 'kibana/public';
 import { useAppDependencies } from '../app_context';
-import { deleteActions } from '../lib/action_connector_api';
 
-export const DeleteConnectorsModal = ({
-  connectorsToDelete,
+export const DeleteModalConfirmation = ({
+  idsToDelete,
+  apiDeleteCall,
   callback,
+  singleTitle,
+  multiplyTitle,
```

Comment:
I think this should be `multipleTitle` rather than `multiplyTitle`.  It's also on alerts.